src/arrays/Arrays.php

```diff
@@ -71,17 +95,27 @@ public function getUniqueValue(array $input): int{
      * ]
      *
      * Make sure the next PHPDoc instructions will be added:
-     * @param array $input
-     * @return array
+     *
+     * @param array $input is non-associative array like: 
+     *                     ['name'=>'nameB', 'tags'=>['tagA,tagB']],
+     *                     where 'tagA' and 'nameA' etc. are arbitrary strings
+     * 
+     * @return array outputs associative array 
+     * of sorted array like: tagN => [nameA, nameB]
      */
-    public function groupByTag(array $input): array{
-        foreach ($input as $key=>$val)
-            $nametags[$val['name']]=$val['tags']; 
-        array_multisort($nametags);
-        // array_count_values($nametags);  
-        var_dump($nametags);
-            // array_multisort 
-        return $nametags;
-    }
+    public function groupByTag(array $input): array
+    {
 
-}
\ No newline at end of file
+        $tagsNames = [];
+        foreach ($input as $val) {
```

Comment:
please try to find different solution

Reply:
Can't find except including sort into internal foreach. IMHO not a good idea becaose sorts always have a slow algorithm.
Have any ideas? 
The problem is I can not get array back from callable function back into main function:
in array_walk, array_walk_recursive, array_map :
 - tried to use static variables
 - tried to use gloval variables
 - tried to use &$pointer notation